<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
     <property name="geometry">
         <rect>
             <x>0</x>
             <y>0</y>
             <width>593</width>
             <height>484</height>
         </rect>
     </property>
     <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
         </sizepolicy>
     </property>
     <property name="windowTitle">
         <string>Form</string>
     </property>
     <widget class="QTableWidget" name="tableWidget">
         <property name="geometry">
             <rect>
                 <x>10</x>
                 <y>50</y>
                 <width>571</width>
                 <height>421</height>
             </rect>
         </property>
     </widget>
     <widget class="QPushButton" name="exit_btn">
         <property name="geometry">
             <rect>
                 <x>10</x>
                 <y>10</y>
                 <width>75</width>
                 <height>23</height>
             </rect>
         </property>
         <property name="text">
             <string>Назад</string>
         </property>
     </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
